_rules/SC1-2-1-video-only-description-track.md

```diff
@@ -0,0 +1,107 @@
+---
+name: Video only element description track
+description: |
+  This rule checks that description tracks that come with non-streaming `video` elements, without audio, are descriptive.
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- Audio output
+- Visual output
+
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible on the page](#visible-on-the-page) where the video doesn't contain audio and does contain a `track` element with a `kind="descriptions"` attribute.
+
+### Expectation
+
+The visual information of each test target is described with a description `track` element that has the same language as the video.
+
+*Note*: Multiple description `track` elements may be useful for different languages, but at least one must match the language of the video.
+
```

Comment:
What about if the video's language is sign language? In what written language should the track be?

Reply:
Wilco and I came to the conclusion we would also pass the language of the track if it was in the same language of the page (because the user is sure to know that language) so we added that. That also solves this issue because the video with sign language can in that case simply have a track element with the same language as the page.